orchestrator.py

```diff
@@ -87,7 +87,8 @@ def __init__(self, broker_ip, port=1883):
             self.battery_adc = AnalogIn(mcp, MCP.P0)
 
             # BCM pin numbering
-            self.connected_led = LED(18)
+            self.connected_led = LED(18)  # Board pin 24
+            self.logging_led = LED(27)  # Board pin 13
```

Comment:
Nice to have: If we can comment or rename these variables to state which colour led these Pins are referring to it might be handy in figuring our what led does what when looking through the code (like we do in `switch.py`).